data/schema/Decision_Point_Group.schema.json

```diff
@@ -0,0 +1,79 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "Decision Points Group schema definition",
+    "$id": "https://github.com/CERTCC/SSVC/tree/main/data/schema/Decision_Point_Group.schema.json",
```

Comment:
I pointed these at the raw github repo URL rather than the certcc.github.io/SSVC... url mentioned in https://github.com/CERTCC/SSVC/pull/342#pullrequestreview-1680382681

Reply:
Shouldn't it be https://raw.githubusercontent.com/CERTCC/SSVC/main/data/schema/Decision_Point_Group.schema.json